src/libraries/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj

```diff
@@ -137,6 +137,7 @@ System.Diagnostics.EventLog</PackageDescription>
                       ReferenceOutputAssembly="false"
                       OutputItemType="TfmRuntimeSpecificPackageFile"
                       PrivateAssets="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.AccessControl\src\System.Threading.AccessControl.csproj" />
```

Comment:
This new dependency will also need to go into the ASP.NET shared framework:
https://github.com/dotnet/runtime/blob/9a02107e78699dbd4b4f2f12277bcc6dbd889e89/src/libraries/NetCoreAppLibrary.props#L223

Looks like it was already present in WindowsDesktop:
https://github.com/dotnet/runtime/blob/9a02107e78699dbd4b4f2f12277bcc6dbd889e89/src/libraries/NetCoreAppLibrary.props#L242